Makefile.toml

```
@@ -102,8 +102,17 @@ cargo d
 [tasks.deadlink]
 script = ['''
 #!/usr/bin/env bash -eux
-cargo deadlinks --check-http --ignore-fragments
+
+mkdir -p target
+set +e
+cargo deadlinks --check-http --ignore-fragments > target/cargo-deadlinks.out
```

Want to check stdout in CI log.

```suggestion
cargo deadlinks --check-http --ignore-fragments | tee target/cargo-deadlinks.out
```